charts/rancher-cis-benchmark/0.2.1/values.yaml

```diff
@@ -0,0 +1,77 @@
+# Default values for rancher-cis-benchmark.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+# if owner is specified, it's used for the name of the configmap for results
+owner: ""
+# userSkipConfigMapName is used to specify the name of cm where user skip info is stored
+userSkipConfigMapName: ""
+# defaultSkipConfigMapName is used to specify the name of cm where default skip info is stored
+defaultSkipConfigMapName: ""
+# notApplicableConfigMapName
+notApplicableConfigMapName: ""
+# overrideBenchmarkVersion is used to override the default benchmark version used for
+# a particular k8s version
+overrideBenchmarkVersion: ""
+
+# when debug=true, the plugin pods sleep for the time specified
+debugMaster: false
+debugWorker: false
+debugTime: "infinity"
+
+sonobuoy:
+  tolerations: []
+
+image:
+  securityScan:
+    repository: rancher/security-scan
+    tag: v0.2.4-rc1
```

Comment:
Need to retag before final RC, sometime next week probably, after this issue is closed.

Reply:
Didn't notice before but the last version in 0.2.0 was 0.1.14: https://github.com/rancher/system-charts/blob/release-v2.6/charts/rancher-cis-benchmark/0.2.0/values.yaml#L30
Why didn't we use 0.2.x before in 0.2.0, is that only for v2 ? @prachidamle @paynejacob

Reply:
Good point @cbron all the changes done in 0.2.0 were for v2. So we have not yet updated the security-scan version for the system-charts since it is used by CIS v1. 

I think most of the changes should be ok, but we need to test CIS v1 before updating the security-scan version. I will run tests and confirm if we are good to update. cc @jakefhyde @paynejacob

Reply:
I see that the Legacy CIS v1 scan is failing with the updated Sonobuoy versions. We possibly need fixes in rancher CIS v1 controller since it looks for earlier sonobuoy labels ("master" Vs "aggregator"). Will work with @jakefhyde on this